src/frontend/src/components/forms/ApiForm.tsx

```diff
@@ -296,6 +297,7 @@ export function ApiForm({ id, props }: { id: string; props: ApiFormProps }) {
       method: method,
       url: url,
       data: data,
+      timeout: props.timeout ?? 5000,
```

Comment:
Comment: 5 seconds seam very long as a default

Reply:
Have adjusted this now, as part of `setApiDefaults`